src/calculateRentalCost.js

```diff
@@ -4,7 +4,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const rentCostForOneDay = 40;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM_DISCOUNT = 20;
+  const LONG_TERM = 7;
+  const SHORT_TERM = 3;
+  let totalCost = rentCostForOneDay * days;
+
+  if (days >= LONG_TERM) {
+    totalCost -= LONG_TERM_DISCOUNT;
+  } else if (days >= SHORT_TERM) {
+    totalCost -= SHORT_TERM_DISCOUNT;
+  }
```

Comment:
```suggestion
  const totalCost = rentCostForOneDay * days;

  if (days >= LONG_TERM) {
    return totalCost - LONG_TERM_DISCOUNT;
  }
  
  if (days >= SHORT_TERM) {
    return totalCost - SHORT_TERM_DISCOUNT;
  }
```
Prefer if with return over if else to simplify later conditions.
<img width="662" alt="image" src="https://github.com/user-attachments/assets/345b7ba4-e8c3-4093-b876-188ef33f82b8">